consensus/polybft/consensus_runtime_test.go

```diff
@@ -782,6 +783,149 @@ func TestConsensusRuntime_calculateStateTxsInput_SecondEpoch(t *testing.T) {
 	polybftBackendMock.AssertExpectations(t)
 }
 
+func TestConsensusRuntime_generateSyncValidatorsDataTxInputRemoved(t *testing.T) {
```

Comment:
maybe better having a combined test where there are added updated and deleted validators